src/content/indices/brazil.mdx

```diff
@@ -0,0 +1,151 @@
+These regional variations reflect differences in pollutants monitored, local climate, population sensitivity, and regulatory frameworks.
+
+
+## Color scale
+
+Brazil’s SMAC IQA calculation consists of five categories: 
+
+<ColorScale variant="Rio de Janeiro" index="brazil" value={true}/>
+
+_Note_: Adapted from https://jeap.rio.rj.gov.br/je-metinfosmac/institucional/docs/entenda_iqa.pdf
+
+
+Brazil’s CETESB  ( São Paulo )IQA calculation consists of five categories with different ranges: 
+
+<ColorScale variant="São Paulo" index="brazil" value={true} />
+
+_Note_: Adapted from https://cetesb.sp.gov.br/ar/padroes-de-qualidade-do-ar/
+
+
+## Methods
+
+1. **Rio De Janeiro - SMAC calculation process**
+Identify Pollutants: First, measure the concentrations of the relevant pollutants SO<sub>2</sub> (µg/m<sup>3</sup>), CO (ppm), PM<sub>10</sub> (µg/m<sup>3</sup>), O<sub>3</sub> (µg/m<sup>3</sup>), NO<sub>2</sub> (µg/m<sup>3</sup>). For each pollutant, the breakpoint table 
+is used to establish thresholds that categorize the levels of pollution into the 5 different categories. 
+
+
+<BreakpointsTable variant="Rio de Janeiro" index="brazil" />
+
+2. **Determine Concentration Ranges** 
+Each pollutant included in the IQA calculation is associated with defined concentration intervals that correspond to specific air quality categories, such as Boa, Regular, Inadeqada, 
+Ma, and Pessima. These ranges serve as the foundation for assigning an air quality index value based on measured pollution levels. 
+
+3. **Use the Formula** 
+For each pollutant, the corresponding IQA sub-index is calculated using a standardized piecewise linear formula. This formula interpolates the pollutant concentration within its 
+applicable range to determine its sub-index value. Apply this formula: 
+
+<div class="latex-container">
+
+<PiecewiseLatexDoc label="IQA" />
+
+</div>
+
+4. The overall IQA value is the highest pollutant sub-index value calculated from the individual pollutant concentration. This value indicates the overall air quality.
+
+5. Classification: The AQI value is then classified into categories based on predetermined ranges.
+
+
+<Admonition type="note">
+
+Note: Sources do not specify a minimum number of pollutants required to calculate the overall IQA in the SMAC methodology. 
+They only mention that specific air pollutants, SO<sub>2</sub>, CO, PM<sub>10</sub>, O<sub>3</sub>, and NO<sub>2</sub> were monitored, but doesn’t detail requirements 
+for the number of pollutants necessary to calculate the overall AQI. 
+
+</Admonition>
+
+
+<AqiCalculator
+  client:only="solid"
+  index="brazil"
+  acronym={frontmatter.indexAcronym}
+/>
+
+
+**Sao Paulo - CETESB calculation process**
+1. In São Paulo, the Air Quality Index (AQI) is calculated using the combined concentrations of six primary pollutants: PM<sub>2.5</sub>, PM<sub>10</sub>, O<sub>3</sub>, NO<sub>2</sub>, SO<sub>2</sub>, and CO. The AQI 
```

Comment:
remove 1,2,3 stlye for a more prose-like approach. use paragraphs to separate steps